portfolio/src/main/webapp/script.js

```diff
@@ -31,4 +31,29 @@ function addRandomQuote() {
 
     console.log(quote);
 
-}
\ No newline at end of file
+}
+
+/** Fetches the hello world and adds it to the page. */
+async function getHelloWorld() {
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+  
+    const helloContainer = document.getElementById('hello-container');
+    helloContainer.innerText = textFromResponse;
+
+  }
+
+async function getShow(){
+     // Send a request to /hello.
```

Comment:
This comment can be removed (it says we're sending a request to /hello, but the code says /show).

Generally we try to not comment what the code does literally because it can become out of sync with the code pretty quickly. To leave a note for a future reader, I'd try to say what we intend to do, something like:

// Get the list of shows from the server.

That comment is much less likely to change over time.

Reply:
Done